Ex2/ManageLibrary.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ex2
+{
+    internal class ManageLibrary
+    {
+        public List<Document> documents { get; set; }
+        public ManageLibrary(List<Document> _documents)
+        {
+            documents = _documents;
+        }
+        public bool addDocument(Document doc)
```

Comment:
The public method should have the first letter in uppercase same as the variables